extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js

```diff
@@ -164,6 +158,22 @@ export class AmpLightboxViewer extends AMP.BaseElement {
     return Promise.resolve();
   }
 
+  /**
+   * @private
+   * @return {!Promise}
+   */
+  initializeLightbox_() {
+    if (!this.carousel_) {
```

Comment:
I think the more AMP-y style is to have the `if` condition be positive then return, and not have an else clause.

i.e.
```js
if (this.carousel_) {
  return Promise.resolve();
}
/* ... */
```